src/plugins/syscalls/win.cpp

```diff
@@ -225,6 +225,15 @@ static event_response_t syscall_cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
 
     wr->tid = info->proc_data.tid;
     wr->w = w;
+    if ( 4 == s->reg_size )
+    {
+        wr->stack_fingerprint = info->regs->rsp + 4 /*return address*/ + 4 * nargs;
+    }
+    else
+    {
+        wr->stack_fingerprint = info->regs->rsp + 8 /*return address*/;
+    }
+    
```

Comment:
It would be really appreciated if you could add short comments about the two calling conventions that are implicitly assumed here. I.e. to explain where the calculation came from.

The rest looks fine.

Reply:
I have added some comments. Is this comment appropriate?  I'm not very good at describing annotations（＾▽＾）.

Reply:
looks good, thanks!